File: CourierAmexClient/src/app/features/company/models/document-type.model.ts
import { BaseEntity } from "@app/models";

export interface DocumentTypeModel extends BaseEntity<number> {
  companyId?: number;
  name: string;
  mask?: string;
  companyName?: string;
}

export const newDocumentType = {
  id: 0,
  companyId: undefined,
  name: '',
  status: 2
};

File: CourierAmexAPI/CourierAmex/Models/DocumentTypeModel.cs
namespace CourierAmex.Models
{
    public class DocumentTypeModel : BaseEntityModel<int>
    {
        public required string Name { get; set; }
        public required int CompanyId { get; set; }
        public string? Mask{ get; set; }
        public string? CompanyName { get; set; }
    }
}
